common/src/main/java/com/google/auto/common/BasicAnnotationProcessor.java

```diff
@@ -358,18 +358,12 @@ private static void findAnnotatedElements(
       default: // do nothing
     }
     for (TypeElement annotationType : annotationTypes) {
-      if (isAnnotationPresent(element, annotationType)) {
+      if (MoreElements.isAnnotationPresent(element, annotationType)) {
```

Comment:
I think this would be more readable if it were statically imported; the class name doesn't add any useful information. Other methods from `MoreElements` are already statically imported.

Reply:
Great observation. Done, thanks.